### DISCUSSION TITLE
Is it safe to delete generated implant under ~/.sliver/slivers/

### DISCUSSION THREAD

**Discussion by jieliau** [BishopFox/sliver#1825] [General]
Thanks guys. I am just on this awesome C2 framework recently. I learned I can issue implants rm to delete implants but the generated executable/shellcode won't be deleted under .sliver/slivers. Am I safe to manually delete the related folder, for example, ~/.sliver/slivers/windows/amd64/FOOLISH_DEVIANCE after I issue implants rm command? Or this might be the space to improve?

**Comment by moloch--** [comment]
Yes, you can safely delete it, we mostly keep these directories around for debugging purposes. We probably should clean these up with the `implants rm` command.
